src/cpu/core_dynrec/risc_ppc64le.h

```diff
@@ -752,7 +754,7 @@ static void gen_run_code(void)
 	IMM_OP(19, 0x14, 0, 16<<1);          // blr
 
 	// trampoline to call get_CF()
-	getCF_glue = cache.pos;
+	getCF_glue = (uint8_t*)cache.pos;
```

Comment:
Suggest updating `getCF_glue` type to:

```
static const uint8_t* getCF_glue = nullptr;
```

So this is a 1:1 assignment without needing to strip-the-const from the pointer.

Reply:
done